### DISCUSSION TITLE
Domain-Level Access(Roles) Control

### DISCUSSION THREAD

**Discussion by forastieri** [open-metadata/OpenMetadata#15279] [Ideas]
Hello,
I would like to suggest an implement domain-level access roles.

The idea is to allow teams, which are associated with a specific domain/team, to have editing roles for objects within their domain. This means a team could edit metadata objects, such as tables, schemas, etc., that are part of their assigned domain but would not have the permission to edit objects in other domains. This approach would help in maintaining cleaner separation of concerns and improve security by limiting based on domain association

**Comment by ayush-shah** [comment]
Hello @forastieri i think we have similar ticket for the same: https://github.com/open-metadata/OpenMetadata/issues/13644
please add your insights / usecase to that ticket, Thanks.

**Reply by forastieri** [reply]
Cool, it's similar, but just to add that it would be an idea for editing roles and not just for viewing. I want to allow the user to view other domains but only edit in their own.
